.github/workflows/unset-environment-new-cli.yml

```
@@ -56,38 +56,38 @@ jobs:
     - shell: bash
       run: |
         CPP_DB="${{ fromJson(steps.analysis.outputs.db-locations).cpp }}"
-        if [[ ! -d "$CPP_DB" ]] || [[ ! "$CPP_DB" == "${RUNNER_TEMP}/customDbLocation/*" ]]; then
-          echo "Did not create a database for CPP, or created it in the wrong location." \
+        if [[ ! -d "$CPP_DB" ]] || [[ ! "$CPP_DB" == "${RUNNER_TEMP}/customDbLocation/"* ]]; then
+          echo "::error::Did not create a database for CPP, or created it in the wrong location." \
             "Expected location was '${RUNNER_TEMP}/customDbLocation/*' but actual was '${CPP_DB}'"
           exit 1
         fi
         CSHARP_DB="${{ fromJson(steps.analysis.outputs.db-locations).csharp }}"
-        if [[ ! -d "$CSHARP_DB" ]] || [[ ! "$CSHARP_DB" == "${RUNNER_TEMP}/customDbLocation/*" ]]; then
-          echo "Did not create a database for C Sharp, or created it in the wrong location." \
+        if [[ ! -d "$CSHARP_DB" ]] || [[ ! "$CSHARP_DB" == "${RUNNER_TEMP}/customDbLocation/"* ]]; then
+          echo "::error::Did not create a database for C Sharp, or created it in the wrong location." \
             "Expected location was '${RUNNER_TEMP}/customDbLocation/*' but actual was '${CSHARP_DB}'"
           exit 1
         fi
         GO_DB="${{ fromJson(steps.analysis.outputs.db-locations).go }}"
-        if [[ ! -d "$GO_DB" ]] || [[ ! "$GO_DB" == "$${RUNNER_TEMP}/customDbLocation/*"" ]]; then
-          echo "Did not create a database for Go, or created it in the wrong location." \
+        if [[ ! -d "$GO_DB" ]] || [[ ! "$GO_DB" == "${RUNNER_TEMP}/customDbLocation/"* ]]; then
+          echo "::error::Did not create a database for Go, or created it in the wrong location." \
             "Expected location was '${RUNNER_TEMP}/customDbLocation/*' but actual was '${GO_DB}'"
           exit 1
         fi
         JAVA_DB="${{ fromJson(steps.analysis.outputs.db-locations).java }}"
-        if [[ ! -d "$JAVA_DB" ]] || [[ ! "$JAVA_DB" == "${RUNNER_TEMP}/customDbLocation/*" ]]; then
-          echo "Did not create a database for Java, or created it in the wrong location." \
+        if [[ ! -d "$JAVA_DB" ]] || [[ ! "$JAVA_DB" == "${RUNNER_TEMP}/customDbLocation/"* ]]; then
+          echo "::error::Did not create a database for Java, or created it in the wrong location." \
             "Expected location was '${RUNNER_TEMP}/customDbLocation/*' but actual was '${JAVA_DB}'"
           exit 1
         fi
         JAVASCRIPT_DB="${{ fromJson(steps.analysis.outputs.db-locations).javascript }}"
-        if [[ ! -d "$JAVASCRIPT_DB" ]] || [[ ! "$JAVASCRIPT_DB" == "${RUNNER_TEMP}/customDbLocation/*" ]]; then
-          echo "Did not create a database for Javascript, or created it in the wrong location." \
+        if [[ ! -d "$JAVASCRIPT_DB" ]] || [[ ! "$JAVASCRIPT_DB" == "${RUNNER_TEMP}/customDbLocation/"*]]; then
```

Fixes a bash syntax error.

```suggestion
        if [[ ! -d "$JAVASCRIPT_DB" ]] || [[ ! "$JAVASCRIPT_DB" == "${RUNNER_TEMP}/customDbLocation/"* ]]; then
```